src/test/java/com/wikia/webdriver/testcases/desktop/adstests/TestAdsRepeatableIncontentBoxadMobileWiki.java

```diff
@@ -23,7 +23,7 @@
 
   private static final String
       AD_INFO_PATTERN
-      = "https?://.*wikia-services\\.com.*kv_pos=incontent_boxad.*kv_rv=4.*";
+      = "https?://.*wikia-services\\.com.*kv_pos=incontent_boxad.*kv_rv=1.*";
```

Comment:
I believe we should check some rv values > 1

Reply:
4 doesn't exist here. I've changed it to higher value